### ISSUE TITLE
Q2: Batch Normalization (34%)

### ISSUE THREAD

**Issue by zheedong** [zheedong/Stanford_CS231n_assignment_2021#7]
In notebook BatchNormalization.ipynb you will implement batch normalization, and use it to train deep fully connected networks.
